packages/calcite-components/src/components/radio-button/radio-button.tsx

```diff
@@ -94,6 +100,11 @@ export class RadioButton
   /** When `true`, the component is not displayed and is not focusable or checkable. */
   @Prop({ reflect: true }) hidden = false;
```

Comment:
Sidebar: we should revisit custom `hidden` props since it's a reflected prop on `HTMLElement`.